react/react1/week1/react-week1/src/App.js

```diff
@@ -0,0 +1,20 @@
+const Todos = [
```

Comment:
I would not write the array constant with uppercase letter as at will cause confusion about what is a component, and what is not. Treat variables in React like regular JavaScript variables.